guide/src/sextant/inspect.md

```diff
@@ -0,0 +1,87 @@
+# Inspecting an NPK
+
+To get information about an already packed NPK `sextant` provides the `inspect` command.
+
+## Inspecting an NPK with Default Settings
+
+Inspecting an NPK without any additional parameters will show the following information:
+
+- Listing of files contained in the NPK
+- Contents of manifest.yaml
+- Contents of signature.yaml
+- Listing of files contained in the compressed squashfs image (`fs.img`) within the NPK
+
+For example, inspecting the `hello` example container gives the following output:
+
+```markdown
+$ sextant inspect target/northstar/repository/hello-0.0.1.npk 
+# inspection of 'target/northstar/repository/hello-0.0.1.npk'
+## NPK Content
+signature.yaml
+manifest.yaml
+fs.img
+
+## manifest.yaml
+---
+name: hello
+version: 0.0.1
+init: /hello
+uid: 1000
+gid: 1000
+env:
+  HELLO: northstar
+mounts:
+  /lib:
+    host: /lib
+  /lib64:
+    host: /lib64
+  /system:
+    host: /system
+io:
+  stdout:
+    log:
+      level: DEBUG
+      tag: hello
```

Comment:
is it possible to include another file into this file? then we could include the manifest from our example and ensure we are always up to date